### ISSUE TITLE
Update weights of model_old to model_new

### ISSUE THREAD

**Issue by suryapa1** [ThilinaRajapakse/simpletransformers#1095]
**Is your feature request related to a problem? Please describe.**
1. Created BertSequntialClassifier with base as pre-trained bert model , and trained on batch of training data and save as model_old
2.1. Created another model (model_new1) with base model as "model_old", and trained with new set of batch data 
2.2 Created another model (model_new2) with base model as "pretrained BERT models", and trained with new set of batch data

Btw, I have updated "__init__" method of BertForSequenceClassification so that except for the Linear classifier layer, bert layers are frozen.

       for param in self.bert.parameters():
            param.requires_grad = False

How to update weights for (2.2), so that it represents learning of batch1 & batch2 ??
How to update weights for (2.1), so that it represents learning of batch1 & batch2 ??

/Code

model = ClassificationModel("bert", "bert-base-uncased")
model.train_model(data1)
output savded in "model_output1" directory

model_new1 = ClassificationModel("bert", "model_output1/")
model_new1.train_model(data2)

model_new2 = ClassificationModel("bert", "bert-base-uncased")
model_new2.train_model(data2)

Update weights of modelnew2 to reflects "model" weights ??? Expected similar to this, but it is not the right example to point out, however, it is not working based on class variables
/Sample code as pseudo-code: take weights of last layers and save it to actual model

    def update_gan_arc(self,gen_v1, gen):
        for i in range(len(gen_v1.layers) - 1):
            if gen_v1.layers[i].get_config().get("name") == "embedding":
                last_layer = gen.get_layer(name="embedding").get_weights()
                temp_final_weights = []
                labels_to_add = 1
                for i in range(len(last_layer)):
                    temp_dense = needed_arr = np.random.rand(1, last_layer[i].shape[1])
                    temp_weights = np.concatenate((last_layer[i], temp_dense), axis=0)
                    temp_final_weights.append(temp_weights)
                gen_v1.get_layer(name="embedding").set_weights(temp_final_weights)
            else:
                gen_v1.layers[i].set_weights(gen.layers[i].get_weights())
        return gen_v1

**Comment by ThilinaRajapakse** [comment]
Isn't that just `model_new2 = ClassificationModel("bert", "model_output1/")`?
